src/core/thermostat.cpp

```diff
@@ -209,8 +247,17 @@ void thermo_heat_up()
 {
   int j;
   if(thermo_switch & THERMO_LANGEVIN) {
+#ifndef PARTICLE_ANISOTROPY
     langevin_pref2_buffer          = langevin_pref2;
     langevin_pref2 *= sqrt(3);
```

Comment:
I just ran a parameter sweep over kT and gamma... the diffusion constant is wrong by a factor of sqrt(3)...